tests/antares/services/api_services/test_area_api.py

```diff
@@ -327,3 +329,97 @@ def test_create_misc_gen_success(self):
             self.area.create_misc_gen(series=self.matrix)
 
             assert mocker.request_history[0].url == expected_url
+
+    def test_read_areas(self):
+        # -web - recette.rte - france
+        study_id_test = "248bbb99-c909-47b7-b239-01f6f6ae7de7"
+        url = f"https://antares.com/api/v1/studies/{study_id_test}/areas"
+        ui_url = url + "?ui=true"
+        url_thermal = url + "/zone/clusters/thermal"
```

Comment:
Introduce a variable `area_id` = "zone" and replace it to make it clearer.
You could also use this variable inside `json_ui` and line 393 when you create the Area.